docs/Contributing/Feature Requests/Feature_requests.md

```diff
@@ -0,0 +1,6 @@
+## Feature requests
+
+Feature requests are welcome. But take a moment to find out whether your idea
+fits with the [scope and aims](https://atos.readthedocs.io/en/latest/) of the project. It's up to *you* to make a strong
+case to convince the project's developers of the merits of this feature. Please
```

Comment:
This text is missing an explanation of what you have to do practically to submit a feature request. Maybe there can be a link to issues.md?

Reply:
Or maybe the files can me merged/restructured. This file is very small.

Reply:
Yes this could be merged. I was thinking that a feature request should be handled separate from an issues/and bugs as it does not require the same submission of information etc. We might also have a different process of how we handle it. But as you say, before we define something like that, as it stands now it doesn't include a lot of extra information